### PR TITLE
Add three 2020 Clojure conferences

### PR DESCRIPTION
**Pull request by aviflax**

### CODE REVIEW

File: conferences/2020/clojure.json [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+  {
+    "name": ":clojureD 2020",
+    "url": "https://clojured.de/",
+    "startDate": "2020-02-29",
+    "endDate": "2020-02-29",
+    "city": "Berlin",
+    "country": "Germany",
+    "cfpUrl": "",
+    "cfpEndDate": "",
+    "twitter": "clojuredconf"
```
Comment (prigara) [inline conferences/2020/clojure.json]: Can you please add `@` before the Twitter handle? Thanks!
(The tests are failing without it right now; will check later if it's actually something we should require)

Comment (aviflax) [reply conferences/2020/clojure.json]: Done!

**Comment by prigara** [comment]
@aviflax thank you! :)

**Comment by aviflax** [comment]
You’re welcome!

**Pull request merged**